src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $color-bg-white;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  box-sizing: border-box;
+  position: relative;
+  padding: 16px;
+
+  &__image {
+    display: block;
+    background-image: url(../images/imac.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 20px;
+  }
+
+  &__title {
+    color: $color-text-dark;
+    font-weight: 500;
+    font-size: $font-size-medium;
+    line-height: 18px;
+    margin-top: 20px;
+  }
+
+  &__product-code {
+    font-size: $font-size-small;
+    font-weight: 400;
+    line-height: $font-size-large;
+    color: $color-text-light;
+    margin: 4px 0 16px;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 166px;
+    height: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    color: $color-text-dark;
+    font-size: $font-size-small;
+    text-align: right;
+    width: 53px;
+    line-height: $font-size-large;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 166px;
+    height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    color: $color-text-light;
+    font-size: $font-size-medium;
+    line-height: 16px;
+  }
+
+  &__price-numbers {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__link {
+    text-transform: uppercase;
```

Comment:
The `&__link` class has a hard-coded width and height. If this is a button, consider making it more flexible by using padding and min-width instead of a fixed width and height to accommodate different text lengths and screen sizes.